src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/AbstractFormToolbarAction.js

```diff
@@ -37,4 +37,8 @@ export default class AbstractFormToolbarAction {
     getToolbarItemConfig(): ?ToolbarItemConfig<*> {
         throw new Error('The getToolbarItemConfig method must be implemented by the sub class!');
     }
+
+    destroy() {
```

Comment:
@danrot we had in the past projects also stores inside the ListToolbarActions, think it would be good and consistent to also have there the `destroy` function. What do you think?